Lib/test/test_os.py

```diff
@@ -113,6 +113,19 @@ def test_getcwd(self):
         cwd = os.getcwd()
         self.assertIsInstance(cwd, str)
 
+    def test_getdtablesize(self):
+        curr_system = platform.system()
+        if curr_system == 'Windows':
```

Comment:
Please remove this code path, and instead skip the test if the function doesn't exist.

You can put such decorator on the test method:

```
@unittest.skipUnless(hasattr(os, 'getdtablesize'), 'need os.getdtablesize()')
```